stablehlo/dialect/StablehloOps.td

```diff
@@ -85,12 +85,20 @@ def StableHLO_IotaOp : StableHLO_Op<"iota", [NoSideEffect]> {
   let summary = "Iota operator";
   let description = [{
     Creates a rank 1 array of values starting at zero and incrementing by one.
+
+    Example:
+
+    ```mlir
+    %0 = mhlo.iota dim = 1 : tensor<1x10xf32>
```

Comment:
I knew I missed something! This should say `stablehlo.iota` (and similar feedback for other places below).